### PR TITLE
Rdm 9113 1

### PR DESCRIPTION
**Pull request by skandala-cgi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-9113

### Change description ###

Migrate DataStore to use HMCTS Gradle Java Plugin

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by MSancaktutar — APPROVED** [review]
LGTM but better to get a second review.

@banderous  - Could you pls give this a quick glance? This is a huge change and it's important to make sure it lands correctly.

File: build.gradle
Comment (banderous) [inline build.gradle:135]: should this be here?

File: build.gradle
Comment (banderous) [inline build.gradle:53]: PMD is not used or required by the gradle plugin, is this change coincidental?

**Review by banderous — APPROVED** [review]

File: build.gradle
Comment (MSancaktutar) [reply build.gradle:135]: Well, it looks like it's just missed.
@skandala-cgi, could you please remove checkstyle plugin traces entirely?

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Please remove all traces of Checkstyle plugin, and for now retain PMD.
PMD will be removed under a separate ticket from other apps altogether in a consistent plan.

File: build.gradle
Comment (MSancaktutar) [inline build.gradle:136]: wouldn't this be removed as well?

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
still some trace of checkstyle perhaps

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Comment by VamshiRudrabhatlaHMCTS** [comment]
This ticket is verified and tested successfully. The check styles were successfully verified from the logs as the part of new HMCTS Java Gradle plug in. Sanity tests (CCD-UI) satisfied. Happy to merge this ticket.

**Pull request merged**
